changelogs/fragments/2935-lvol-support_check_mode_thinpool.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - lvol - support check_mode on thinpool (https://github.com/ansible-collections/community.general/issues/2934).
```

The module already supports `check_mode` - it just doesn't care about it for `thinpool`.
```suggestion
  - lvol - honor ``check_mode`` on thinpool (https://github.com/ansible-collections/community.general/issues/2934).
```